.circleci/config.yml

```diff
@@ -1,36 +1,26 @@
 version: 2.1
 
 orbs:
-  ci-caching: jobcloud/ci-caching@3.1
-  ci-php: jobcloud/ci-php@2.4
+  ci-caching: jobcloud/ci-caching@3.1.1
+  ci-php: jobcloud/ci-php@2.4.0
```

Comment:
There is no need to add patch version, it will receive all patch updates, see https://circleci.com/docs/2.0/orb-concepts#semantic-versioning

Reply:
I've changed it back.